app/main.ipynb

```diff
@@ -23,24 +19,42 @@
     "Also check for data correctness (salary & salary $K)."
    ],
    "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
+    "collapsed": false
    }
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 80,
    "outputs": [],
    "source": [
-    "# write your code for task 0 here"
+    "df = pd.read_csv(\"../data/adult.csv\")"
    ],
    "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%%\n"
-    }
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 81,
+   "outputs": [],
+   "source": [
+    "indexForDrop = df[(df[\"workclass\"] == \"?\")].index\n",
+    "df.drop(indexForDrop , inplace=True)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 82,
+   "outputs": [],
+   "source": [
+    "indexForDrop = df[(df[\"native-country\"] == \"?\")].index\n",
+    "df.drop(indexForDrop , inplace=True)"
```

Comment:
Also check for data correctness (salary & salary $K).
Check salary and salary $K colums. If value of salary column is "<=50K", then salary $K value should be <= 50.
The same for >50K